src/test/test-compress-benchmark.c

```diff
@@ -160,13 +160,13 @@ int main(int argc, char *argv[]) {
 
         NULSTR_FOREACH(i, "zeros\0simple\0random\0") {
 #if HAVE_XZ
-                test_compress_decompress("XZ", i, compress_blob_xz, decompress_blob_xz);
+                test_compress_decompress("xz", i, compress_blob_xz, decompress_blob_xz);
 #endif
 #if HAVE_LZ4
-                test_compress_decompress("LZ4", i, compress_blob_lz4, decompress_blob_lz4);
+                test_compress_decompress("lz4", i, compress_blob_lz4, decompress_blob_lz4);
 #endif
 #if HAVE_ZSTD
-                test_compress_decompress("ZSTD", i, compress_blob_zstd, decompress_blob_zstd);
+                test_compress_decompress("zstd", i, compress_blob_zstd, decompress_blob_zstd);
```

Comment:
These changes are not necessary, the strings are used only in the log message. Please drop.